python/lsst/ts/MTMount/mtmount_csc.py

```diff
@@ -111,9 +111,35 @@ def __init__(
         # Task for self.read_loop
         self.read_loop_task = salobj.make_done_future()
 
-        # Most recent Rotator Application telemetry sample,
-        # or None if camera cable wrap not tracking.
-        self.prev_rot_application = None
+        # Most recent Rotator Application telemetry sample information,
+        # or None if no information from the rotator was ever received or if
+        # CCW disabled.
+        self.last_rotator_position = None
+        self.last_rotator_time = None
+
+        # CCW - Rotator synchronization limits.
+        # The goto limit is the distance between the CCW and the Rotator demand
+        # position after which the CCW will blindly go to the demand, as long
+        # as the distance between the ccw and rot are less then half the max
+        # limit.
```

Comment:
What is the significance of "half the max limit"? What if the distance is greater than that? Surely even then we should be slewing as fast as we can in a desperate attempt to prevent the interlock from firing? Or do you just give up at that point?

Reply:
If the distance between rotator and ccw is greater than "half the max limit" the CCW will be kept in a desperate attempt to follow the rotator... If not, it wil go to the demand position.

Reply:
In that case I'm afraid I don't understand the text at all. Are you saying that rotator follows demand (rather than trying to "catch up") if and only if both of the following are true: |current CCW - rotator demand| < goto limit AND |current CCW - current rotator| < max limit?